avm/res/document-db/database-account/main.bicep

```diff
@@ -885,11 +898,17 @@ type sqlDatabaseType = {
     @description('Optional. Default to 400. Request Units per second. Will be ignored if autoscaleSettingsMaxThroughput is used.')
     throughput: int?
 
-    @description('Optional. The unique key policy configuration containing a list of unique keys that enforces uniqueness constraint on documents in the collection in the Azure Cosmos DB service.')
+    @description('Optional. [DEPRECATED] The unique key policy configuration containing a list of unique keys that enforces uniqueness constraint on documents in the collection in the Azure Cosmos DB service.')
```

Comment:
What is the reason for deprecating in favor of a new name? Is it PSRule related by any chance?

Reply:
That's the only thing keeping me from all greens and no warnings (I have all greens on deployment and 1 warning now)

Reply:
BTW, I'm open to any other ideas. I just saw the requirement to have all green build with no warnings and this PSRule warning has been tricky to resolve.

Reply:
Yes @seesharprun I agree and that's the same I'd have suggested. You followed the deprecation process in the best way and thanks a lot for that. I see that more applicable after v1.0.0 is released, but that's only my personal opinion. Other maintainers may have a different view. However for the time being, having the parameter name listed in the ps-rule.yml file is the best option. Thanks for the great contribution!

Reply:
To be completely honest I am more keen on just removing the deprecated parameter/code/test/whatever than accumulating it until we are allowed to bump the major version. We have in our AVM notes that the versions 0.*.* can contain breaking changes in any form.

![image](https://github.com/user-attachments/assets/5c06be6f-c68b-4044-81b9-d4fef69db133)

Reply:
How about this for a compromise:

We get the bug fix in soon, and do the breaking change in a different PR to rename the property.

Would that work?